README.md

```diff
@@ -142,6 +142,7 @@ toc:
   sublist_class: ''
   item_class: toc-entry
   item_prefix: toc-
+  anchor_id_url_encoded: false
```

Comment:
You don't need to make it configurable.
I decided all IDs are encoded by default 👍

Reply:
Where is the encoding happen? I don't saw the line in master branch. Should we drop the whole PR, and what about the issues?

Reply:
Oh, sorry, the encoding behavior haven't merged yet on master.

I mean that:

- `anchor_id_url_encoded: true` should be a default behavior
- Remove `anchor_id_url_encoded` setting from Configuration ( since I think it should not be configurable)